aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -2732,7 +2732,7 @@ Tensor linalg_cond(const Tensor& self, const optional<Scalar>& opt_ord) {
 
   // If ord == None or ord == ±2
   if (std::abs(ord.toDouble()) == 2.0) {
-    auto singular_values = std::get<1>(at::svd(self));
+    auto singular_values = at::linalg_svdvals(self);
```

Comment:
This diff is most likely the reason for the XLA tests failure. https://github.com/pytorch/pytorch/issues/71964